test/test-server/srcm_handlers.go

```diff
@@ -18,7 +18,7 @@ const (
 	idRegion1          = "sgreg-1"
 	packageTypeRegion1 = "advanced"
 	cloudRegion1       = "aws"
-	nameRegion1        = "us-east-2"
+	nameRegion1        = "us-west-2"
```

Comment:
I think we should either keep this as `east` to match `displayNameRegion1 = "Ohio (us-east-2)"`, or change the display name.